### DISCUSSION TITLE
Do I have Superfast enabled properly?

### DISCUSSION THREAD

**Discussion by Aston77** [enzo1982/freac#486] [Help & Support] [answered]
I've been trialing fre:ac as a potentially faster alternative to ffmpeg on my 24-core 32-thread machine but my process run times show it actually being considerably slower. I am using the most recent builds of both and generating similar-quality output files. 

Is there something I missing in my command that is preventing the speed boost from fre:ac ?

`ffmpeg -i input.mp4 -codec:a libmp3lame -qscale:a 2 output.mp3` = 29sec

`freaccmd --encoder=lame -q 2 --superfast --threads=auto input.mp4` = 48sec

**Comment by enzo1982** [comment]
Your command line looks correct. I have a 16 core, 24 thread system and get the following results for a large album with the same command lines on Windows:

    fre:ac 1.1.7: 12 sec
    FFmpeg 6.0: 70 sec

So I'm not sure what's causing the slow conversion on your system.

Which operating system are you using? And are you using an up-to-date version of fre:ac?

**Reply by Aston77** [reply]
Win 11 and I have the exact same versions of the above packages installed.

I was however converting one 90min m4a file instead of an album of separate tracks. Could that be part of the difference?

**Reply by Aston77** [reply, marked as answer]
![image](https://user-images.githubusercontent.com/78501124/235772262-82b65ae5-93ae-4c97-b343-19c153c9c789.png)

Looking further into it it appears that the problem could be my input files quality wasn't equal when I started to dig further into the details. 

When I controlled that variable using the freac version ran in ~9sec and the ffmpeg ran in ~28sec.

Sorry for the confusion.

**Reply by enzo1982** [reply]
Alright, thank you for reporting back!
